src/test/test/opinion/rules/ShouldRuleTest.java

```diff
@@ -0,0 +1,72 @@
+package opinion.rules;
+
+import opinion.rules.impls.ShouldRule;
+import opinion.sentences.Sentence;
+import org.junit.Assert;
+import org.junit.Test;
+import proto.RuleResultOuterClass;
+
+import java.util.List;
+
+
+public class ShouldRuleTest {
+
+    private RuleEngine ruleEngine = new RuleEngine(getTestRuleList());
+    private SentenceRule rule = RuleFactory.getRule("should");
+
+    proto.RuleListOuterClass.RuleList getTestRuleList() {
+        proto.RuleListOuterClass.Rule testRule = proto.RuleListOuterClass.Rule.newBuilder()
+                .setRuleName("should")
+                .build();
+        proto.RuleListOuterClass.RuleList ruleList = proto.RuleListOuterClass.RuleList.newBuilder()
+                .addRules(testRule)
+                .build();
+        return ruleList;
+    }
+
+    @Test
+    public void testGetRuleName() {
+        proto.RuleListOuterClass.Rule shouldRule = proto.RuleListOuterClass.Rule.newBuilder()
+                .setRuleName("should")
+                .build();
+
+        Assert.assertEquals(shouldRule.getRuleName(), "should");
```

Comment:
This test doesn't appear to be testing the `ShouldRule` class. What does this test cover/what is its point?